test/RestrictedNORI-transfer.test.ts

```diff
@@ -0,0 +1,531 @@
+import { BigNumber } from 'ethers';
+
+import {
+  expect,
+  advanceTime,
+  batchMintAndListRemovalsForSale,
+} from '@/test/helpers';
+import {
+  setupTestLocal,
+  restrictRemovalProceeds,
+  compareScheduleDetailForAddressStructs,
+  compareScheduleSummaryStructs,
+  SECONDS_IN_10_YEARS,
+} from '@/test/helpers/restricted-nori';
+
+describe('RestrictedNORI transferring', () => {
+  describe('success', () => {
+    describe('safeTransferFrom', () => {
+      // eslint-disable-next-line jest/expect-expect -- assertions are in helper function
+      it('should transfer some tokens to another account', async () => {
+        const removalDataToList = [
+          {
+            amount: 100,
+            vintage: 2018,
+          },
+        ];
+        const testSetup = await setupTestLocal({});
+        const { rNori, hre } = testSetup;
+        const { listedRemovalIds, projectId } =
```

Comment:
not for this pr, but a note for the future: tweak `userFixtures` to allow specifying removal state for a supplier. Makes it easier to separate test setup logic from test logic.